app/Http/routes.php

```diff
@@ -27,6 +27,8 @@
     Route::resource('competitions', 'CompetitionsController');
 
     // Waiting rooms routes.
+    Route::get('waitingrooms/{waitingrooms}/split', 'WaitingRoomsController@showSplitForm')->name('split');
+    Route::post('waitingrooms/{waitingrooms}/split', 'WaitingRoomsController@split')->name('split');
```

Comment:
did it not work using `['as' => 'split', 'use' => 'WaitingRoomsController@split']` etc? I feel this is more readable as to naming the route, rather than calling a name function. But if it doesn't bug anyone else, then not a huge deal...